lib/src/package_config_json.dart

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import "dart:convert";
+import "dart:io";
+import "dart:typed_data";
+
+import "package:path/path.dart" as path;
+
+import "package_config_impl.dart";
+import "packages_file.dart" as packagesfile;
```

Comment:
nit: https://dart.dev/guides/language/effective-dart/style#do-name-import-prefixes-using-lowercase_with_underscores

Reply:
True. I'll have to find a single name then, because I don't like underscores :)